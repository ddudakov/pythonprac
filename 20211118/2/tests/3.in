class C(str):
    num = Num()

print(C().num)
c, d = C("qwer"), C("qweqrqt")
c.num = "qwertyu"
d.num = [i for i in range(5)]
print(c.num+d.num)
c.num = 10890
print(c.num+d.num)


