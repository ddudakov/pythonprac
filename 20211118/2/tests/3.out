0
12
10895
